src/python_bindings/opt_to_py.cpp

```diff
@@ -0,0 +1,35 @@
+#include "opt_to_py.h"
+
+namespace {
+namespace py = pybind11;
+using ConvFunction = std::function<py::object(boost::any)>;
+
+template <typename T>
+std::pair<std::type_index, ConvFunction> normal_conv_pair{
+        std::type_index(typeid(T)),
+        [](boost::any value) { return py::cast(boost::any_cast<T>(value)); }};
+
+template <typename T>
+std::pair<std::type_index, ConvFunction> enum_conv_pair{
+        std::type_index(typeid(T)),
+        [](boost::any value) { return py::cast(boost::any_cast<T>(value)._to_string()); }};
+const std::unordered_map<std::type_index, ConvFunction> converters{
```

Comment:
```suggestion
std::unordered_map<std::type_index, ConvFunction> const converters{
```
I wonder why clang-format didn't catch this